docs/howto/add_relationship_links.md

```diff
@@ -0,0 +1,132 @@
+[Back to Guides](../README.md)
+
+# How to add relationship links
+
+AMS offers you many ways to add links in your JSON, depending on your needs.
+
+The following examples are in Rails but the logic is the same for any framework
```

Comment:
Is this true? none of the controllers (or even the serializers) have a superclass.